src/main/java/org/broadinstitute/hellbender/engine/ReferenceFileSource.java

```diff
@@ -35,8 +48,7 @@
      * @param fastaFile reference fasta file
      */
     public ReferenceFileSource(final File fastaFile) {
-        // Will throw a UserException if the .fai and/or .dict are missing
-        reference = CachingIndexedFastaSequenceFile.checkAndCreate(Utils.nonNull(fastaFile));
+        this(fastaFile.toPath());
```

Comment:
This will throw an NPE if the `fastaFile`is null. Better: `this(Utils.nonNull(fastaFile).toPath())`

Reply:
Good call, done.